Source/WebCore/Modules/mediastream/RTCDataChannel.cpp

```diff
@@ -95,6 +95,10 @@ RTCDataChannel::RTCDataChannel(ScriptExecutionContext& context, std::unique_ptr<
 {
 }
 
+RTCDataChannel::~RTCDataChannel()
```

Comment:
Could this not be ` = default` (ideally in the header)?

Reply:
No, I tried that and it didn't work.

Reply:
Annoying!

Reply:
I suppose you have tried also `= default` in the `.cpp` file and that didn't work either?

Reply:
Hm that seems to work actually!